OrbitGl/App.cpp

```diff
@@ -975,11 +974,16 @@ void OrbitApp::SelectTextBox(const TextBox* text_box) {
 
 void OrbitApp::UpdateSamplingReport() {
   if (sampling_report_ != nullptr) {
-    sampling_report_->UpdateReport();
+    Capture::capture_data_.UpdateSamplingProfiler();
+    sampling_report_->UpdateReport(Capture::capture_data_.GetSamplingProfiler(),
+                                   Capture::capture_data_.GetCallstackData());
   }
 
   if (selection_report_ != nullptr) {
-    selection_report_->UpdateReport();
+    SamplingProfiler selection_profiler = selection_report_->profiler();
+    selection_profiler.ProcessSamples(*Capture::capture_data_.GetSelectionCallstackData());
+    selection_report_->UpdateReport(std::move(selection_profiler),
+                                    Capture::capture_data_.GetSelectionCallstackData());
```

Comment:
I think that it would be enough to create an empty profiler in first line (`ProcessSamples` clears data as the first step anyway).

Reply:
Done. Thanks for spotting this.